src/lib/DiscordWebhook/sendStats.ts

```diff
@@ -11,11 +11,16 @@ export default function sendStats(bot: Bot, forceSend = false, steamID?: SteamID
     const optDW = bot.options.discordWebhook;
     const botInfo = bot.handler.getBotInfo;
     const trades = stats(bot);
-    const profits = profit(bot);
+    const profits = profit(bot, Date.now() - 86400000);
```

Comment:
This in milliseconds right? Should be divided by 1000 and round the value so it will match with pollData timestamps (Unix, in seconds).